.github/workflows/ci.yml

```diff
@@ -0,0 +1,112 @@
+name: continuous-integration
+on: [push]
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        py: [ 3.6, 3.7, 3.8, 3.9 ]
+        CC: [ gcc ]
+
+        # Check macos too
+        include:
+          - os: macos-latest
+            py: 3.7
+
+    steps:
+        - uses: actions/checkout@v2
+
+        - name: Set up Python ${{ matrix.py }}
+          uses: actions/setup-python@v2
+          with:
+            python-version: ${{ matrix.py }}
+
+        - name: Install GFortran macOS
+          if: matrix.os == 'macos-latest'
+          run: |
+            brew cask install gfortran || true
```

Comment:
Could leave a comment here about the || true bit.  Since you got it from me, I can say that this is because brew returns a failure exit code if the thing is already installed and current (which would often be the case if you used a cache, which you aren't doing yet).  So || true means this statement always is considered to succeed.  If there was a real failure in brew installation, then something later will fail instead.

Reply:
Done